### ISSUE TITLE
Excluded urls in the settings reported as broken links

### ISSUE THREAD

**Issue by allison-soo** [central-queensland-uni/moodle-tool_crawler#24]
I added https://tanzecampus-my.sharepoint.com/ and https://tanzecampus.sharepoint.com/ to "Exclude external URL's" to exclude the following urls, with no success.

https://tanzecampus-my.sharepoint.com/_layouts/15/MySite.aspx?MySiteRedirect=AllDocuments
https://tanzecampus.sharepoint.com/_layouts/15/sharepoint.aspx

I tried to put in full url and this didn't help either. Both urls are still reported as broken links.

**Comment by brendanheywood** [comment]
@allison-soo can you please dump a screen shot of the exclude url settings and/or an exact copy of the textarea content. I have a suspicion that there is some sort of regex bug. Also check that the links in the page aren't http (vs https)

**Comment by allison-soo** [comment]
@brendanheywood Here are the screenshots.
![broken links in staging](https://cloud.githubusercontent.com/assets/28668582/26039657/8e864896-3971-11e7-9032-c588e70952cb.JPG)
![link crawler settings in staging](https://cloud.githubusercontent.com/assets/28668582/26039658/8e999c20-3971-11e7-994a-a6ed4a303829.JPG)

**Comment by brendanheywood** [comment]
Perhaps try and combine all 4 lines into one much simpler pattern such as just "sharepoint.com"

**Comment by allison-soo** [comment]
if just "sharepoint.com", does it mean it will exclude http://www.sharepoint.com and also all urls under it?

**Comment by brendanheywood** [comment]
Yes. I am pretty sure that every url under that gets bounced to the sharepoint login page anyway and isn't visible to the robot anyway so blocking the whole domain is desirable.
